staging/src/k8s.io/kubectl/pkg/cmd/kuberc/kuberc.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kuberc
+
+import (
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/yaml"
+)
+
+// Example kuberc.yaml file
+// ---
+//apiVersion: v1alpha1
+//kind: Preferences
+//command:
+//  aliases:
+//    dbprod:
```

Comment:
I'd prefer this way instead;
```yaml
aliases:
   get pods:
     ...
```
`dbprod` should be unique and this way more emphasizes on this requirement.

Reply:
So the thing is that the aliases will only be unique to the subcommand itself and will not be unique globally. As far as I can tell cobra doesn't allow global aliasing, so you would be running commands like `k get dbprod` and not `k dbprod`. See [here](https://github.com/spf13/cobra/issues/1725) for more details.